dmoj/executors/JAVA.py

```diff
@@ -2,9 +2,9 @@
 
 
 class Executor(JavacExecutor):
-    compiler = 'javac9'
-    vm = 'java9'
-    jvm_regex = r'java-9-|openjdk9|oracle-java9'
+    compiler = 'javac'
+    vm = 'java'
+    jvm_regex = r'(?:java-|openjdk)(?:9|1[0-9])'
```

Comment:
```suggestion
    jvm_regex = r'(?:java-|openjdk)(?:9|[1-9][0-9])'
```

Let's just do this so we don't have to do any work when 20 comes around.